virt/virt-4-10-release-notes.adoc

```diff
@@ -117,20 +122,18 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-10-bug-fixes"]
 == Bug fixes
 
+* If you hot-plug a virtual disk and then force delete the `virt-launcher` pod, you no longer lose data. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2007397[*BZ#2007397*])
+
+* In previous releases, if a cloning operation was initiated before the clone source became available, the cloning operation stalled. This is no longer an issue, as the clone authorization does not expire before the cloning operation starts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1855182[*BZ#1855182*])
```

Comment:
My suggestion would be to get rid of the passive voice and rewrite for clarity. Maybe something like this:

````
If you initiate a cloning operation before the clone source becomes available, the cloning operation now completes successfully without using a workaround.
````

Reply:
Done.